lang/libomp/Portfile

```diff
@@ -31,40 +30,42 @@ if { ${subport} eq "libomp-devel" } {
         replaced_by             libomp
     } else {
         conflicts               libomp
-        github.setup            llvm-mirror openmp 7a5b758
-        worksrcdir              openmp-${version}
-        version                 349521
+        # Moved to official llvm github project. It would be a very large 
+        # (100MB+) download, so use the svn interface (!) to github to 
+        # download just the openmp directory. Not using LLVM's own SVN since
+        # they are in the process of migrating to github.
+        fetch.type              svn
+        svn.url                 https://github.com/llvm/llvm-project/trunk/openmp
+        svn.revision            327229
+        # llvm-svn revision; also used for livecheck
+        version                 367070
```

Comment:
This is a non-sensical `version`. At least use a date format (e.g. `2019-07-26`) so it can increase going forward.

Reply:
Is this the build count? AFAIK, no other port uses a version numbering scheme like that.

Reply:
How do you figure? This is the upstream SVN revision number. Now, I’ll give you that the two SVNs here makes it confusing (the upstream SVN and the SVN # from github’s SVN interface), but I’m continuing to use the SVN number from the still-primary upstream SVN as the version number for this (head-tracking) -devel version. (At least until they fully move to GH.

Reply:
Well I suppose if it increases it is fine.